lib/Mojolicious/Guides/Rendering.pod

```diff
@@ -1128,7 +1128,7 @@ applications, plugins make that very simple.
 
   1;
 
-The C<register> method will be called when you load the plugin and to add your
+The C<register> method will be called when you load the plugin; to add your
```

Comment:
oh, crap, I actually agree, I seem to have gone red/green colorblind on this one. to be clear, I prefer the semicolon. sorry for the confusion